### PR TITLE
IT: together-we-are-powerful.po

### PR DESCRIPTION
**Pull request by ffissore**
Fixed wrong hyphen in `diseases.po`, added `together-we-are-powerful.po`

### CODE REVIEW

File: Localization/it-IT/together-we-are-powerful.po [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+msgstr ""
+
+msgid "Satya Nadella"
+msgstr ""
+
+msgid "CEO, Microsoft"
+msgstr ""
+
+msgid "Dr. Lisa Su"
+msgstr ""
```
Comment (tematim) [inline Localization/it-IT/together-we-are-powerful.po]: A lot of msgstr are empty, if there is no equivalent I think you may need to keep original text.

**Review by tematim — CHANGES_REQUESTED** [review]

File: Localization/it-IT/together-we-are-powerful.po (hunk 1 above)
Comment (ffissore) [reply Localization/it-IT/together-we-are-powerful.po]: Is there a guideline on this? The behaviour of gettext is that, when `msgstr` is empty, `msgid` is used, which is to say "if there's no translation, use the original text"

Comment (Tar-Minyatur) [reply Localization/it-IT/together-we-are-powerful.po]: Yes, please don't leave any values empty. The system in the end will just replace any instance of the value in `msgid` with the value in `msgstr`. So...if it's empty the resulting page will be incomplete.

Comment (ffissore) [reply Localization/it-IT/together-we-are-powerful.po]: Fixed

**Review by Tar-Minyatur — APPROVED** [review]
Looks good to me from a technical standpoint. Sadly, I don't speak Italian. ;)

**Pull request merged**